workspaces/nexus-repository-manager/plugins/nexus-repository-manager/src/components/NexusRepositoryManager/NexusRepositoryManager.tsx

```diff
@@ -33,6 +33,9 @@ export function getAssetVariants(component: ComponentXO) {
   );
 }
 
+/**
+ * @public
+ */
 export function NexusRepositoryManager() {
```

Comment:
```suggestion
export const NexusRepositoryManager = () => {
```

Doing this mean that you could then remove the export in the root `index.ts` file and the API Report would be clean as well.